samples/iptcprint.cpp

```diff
@@ -68,5 +68,9 @@ try {
 }
 catch (Exiv2::AnyError& e) {
     std::cout << "Caught Exiv2 exception '" << e << "'\n";
-    return -1;
+    return 1;
```

Comment:
Could we use here `EXIT_FAILURE`? I like to use those exit values macros for making the code more readable:

https://en.cppreference.com/w/cpp/utility/program/EXIT_status

It is also fine for me if you want to merge this directly, and we can do a whole replacement of exit values in another PR